style.css

```diff
@@ -1,10 +1,78 @@
 @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap");
+
+:root {
+  --primary-color: #134dc7;
+  --secondary-color: #172b4d;
+  --default-transition: all 0.3s ease-in-out;
+}
+
 * {
   margin: 0;
   padding: 0;
+  box-sizing: border-box;
 }
 
 body {
-  background-color: #ffffff;
+  background-color: #fff;
   font-family: "Poppins", sans-serif;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 20px;
+}
+
+.brand-name {
+  color: var(--primary-color);
+}
+
+.hamburger-btn {
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  cursor: pointer;
+  transition: var(--default-transition);
+}
+
+.hamburger-btn span {
+  height: 3px;
+  width: 30px;
+  background: var(--primary-color);
+  border-radius: 2px;
+}
+
+.hamburger-btn:hover span {
+  background-color: var(--secondary-color);
+}
+
+main {
+  padding: 10px 30px;
+}
+
+.headline {
+  margin-top: 40px;
+}
```

Comment:
This section doesn't have the template's background image, please add it.

<img width="694" alt="Screen Shot 2022-08-18 at 8 42 12" src="https://user-images.githubusercontent.com/82426682/185409768-b207acff-3578-4af7-8d4b-7827cfcaeda2.png">

Additionally, this content should be aligned with flexbox.

<img width="235" alt="Screen Shot 2022-08-18 at 8 45 29" src="https://user-images.githubusercontent.com/82426682/185410455-ea35454b-bbb0-467a-af59-6a64a499614c.png">